Channels/Insider/release_notes.md

```diff
@@ -2,6 +2,10 @@
 
 Welcome to the April 18th, 2019 release of Accessibility Insights for Windows.
 
+Installation Link: [https://github.com/Microsoft/accessibility-insights-windows/releases/download/v1.1.0837.02/AccessibilityInsights.msi](https://github.com/Microsoft/accessibility-insights-windows/releases/download/v1.1.0837.02/AccessibilityInsights.msi)
+
+Documentation Link: [https://accessibilityinsights.io/docs/en/windows/overview](https://accessibilityinsights.io/docs/en/windows/overview)
```

Comment:
Since documentation will always track Production, having the link here may lead to future confusion. We may want to only include it in the Production version of the file. Thoughts?

Reply:
Two possibilities are readily apparent:
1. We have Insider documentation and Production documentation. The Insider docs would be promoted to production at the same time as the build . If a variant of this is possible, it would be efficient since the docs would match the build and it would involve only the same amount of manual intervention as promoting the build itself.
2. The release notes contain the documentation that differs between the Insider and Production builds. The release notes also contain a caveat that the production docs may not be up-to-date. This involves more manual intervention when the build is finally pushed because the documentation will need to be updated at that time.

Reply:
I suspect that having separate sets of docs is probably a non-starter--we're not staffed for that and it sort of goes contrary to a rapid release rate.

Including abbreviated instructions might be the eventual solution, although in our best James Shore approach, we would probably do this only if there's sufficient customer demand to justify it.

Ultimately, I guess my question was somewhat rhetorical and more about raising awareness than it was really about finding a solution. Our short-term "realistic" options may boil down to (1) point Insider to Production docs or (2) omit the documentation link from Insider. The lowest-friction option is probably to include the link to Production docs, if for no other reason than our Production promotion becomes copying the Insider notes and updating the release date.

---
In reply to: [279404997](https://github.com/Microsoft/accessibility-insights-windows/pull/341#discussion_r279404997) [](ancestors = 279404997)

Reply:
as Dave says, having two sets of docs is a non started. Pointing insider to acccessibilityinsights.io should be fine